docs/getting-started/building-bpfman.md

```diff
@@ -241,6 +241,20 @@ sudo dnf install perl
 sudo apt install perl
 ```
 
+### Install docker
+
+To build the `bpfman-agent` and `bpfman-operator`, `docker` needs to be installed.
```

Comment:
I think we need to make sure builds work with podman as well just to be good net citizens :) I thought they did!

Reply:
For KIND we'll always need docker though

Reply:
But at the very least le't just say "container build tooling" or something like that

Reply:
So I agree that should make sure that builds work with podman, but `make build-images` specifically calls `docker build`. I just hit an issue on my server that took an hour to debug. I had `docker` installed but not `docker buildx`, so `make build-images` was failing. So we definitely have a dependency on docker.